fili-core/src/main/java/com/yahoo/bard/webservice/application/DruidDimensionsLoader.java

```diff
@@ -84,6 +84,54 @@ public DruidDimensionsLoader(
             DimensionDictionary dimensionDictionary,
             DruidWebService druidWebService
     ) {
+        this(
+                druidWebService,
+                buildDimensionList(dimensionDictionary),
+                buildDataSourcesList(physicalTableDictionary)
+        );
+    }
+
+    /**
+     * Builds a list of datasources from the physical tables.
+     * @param physicalTableDictionary  The physical tables to load from.
+     * @return the list of datasources.
+     */
```

Comment:
Generally, it's:

1. Blank line between description and `@` clauses
2. Blank line between `@param` section and the remaining `@` clauses.
3. Optionally a blank line after the `@return` clause (if there is one, and if there are additional clauses)
    - And the `@return` clause should be the next one after the `@param` ones

The only ones that really matter are the 1st and 2nd ones. The 3rd one is optional (ie. if you think it makes it more readable / visible)